addon/services/user-idle.js

```diff
@@ -1,10 +1,9 @@
-import classic from 'ember-classic-decorator';
 import Ember from 'ember';
 import EventManagerService from 'ember-user-activity/services/-private/event-manager';
 import { inject as injectService } from '@ember/service';
 import { cancel, debounce } from '@ember/runloop';
+import { set } from '@ember/object';
```

Comment:
Avoid using `set`. We can use native assignment and tracked properties

Reply:
Done